### ISSUE TITLE
[ENG] Prison of Immol'thar in Dire Maul bugged

### ISSUE THREAD

**Issue by Betraye21** [Atlantiss/BugTracker#3795]
I have destroyed all pillars (or towers, crystals i dont exactly know how to call it), but the protection of the prison of Immol'thar didnt fade.

**Comment by sigmundfreud2016** [comment]
I confirm it. Better turn the prison off completely if its hard to repair pylons. Because now its impossible to finish this quest http://www.wowhead.com/quest=27110/the-madness-within

**Comment by roirraWedorehT** [comment]
[https://github.com/Atlantiss/BugTracker/issues/6244](url)

Confirmed that the quest works fine, deactivate the pylons by standing directly underneath the base and it makes a sound and looks different when it's deactivated, plus the gear symbol no longer appears (unlike when you unsuccessfully deactivate by not standing in the right place). Also we noticed pylon #1 had a targetable/attackable "enemy" by the name of the pylon, and we happened to kill it before deactivating the pylon.

It might not have been necessary but we deactivated them in pylon number order, 1 through 5.
